sos/report/plugins/bcache.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (C) 2021, Canonical ltd
+# Ponnuvel Palaniyappan <ponnuvel.palaniyappan@canonical.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, IndependentPlugin
```

Comment:
You'll need to explicitly import `SoSPredicate` here. That's the reason the `flake8` test is failing.

After that, this LGTM.

Reply:
Done, Thanks!